op-chain-ops/crossdomain/precheck_test.go

```diff
@@ -0,0 +1,119 @@
+package crossdomain
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/ethereum-optimism/optimism/op-bindings/predeploys"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/rawdb"
+	"github.com/ethereum/go-ethereum/core/state"
+	"github.com/ethereum/go-ethereum/trie"
+	"github.com/stretchr/testify/require"
+)
+
```

Comment:
Do you mind adding human readable text to these tests so i can verify against what you say they are supposed to do?

Reply:
Done.